tests/conftest.py

```diff
@@ -298,6 +307,50 @@ def email_target_json():
     return targets
 
 
+@pytest.fixture
+def multiple_click_object():
+    """Return a list of clicks to match the correct number of unique users."""
+    clicks = list()
+    for x in range(1, 2):
```

Comment:
What is the purpose of this `for x in range(1, 2):` statement here?

Reply:
I think it might have been a left over from something I was writing before and forgot to remove. I believe originally I was gonna iterate over the clicks and append, but decided against it and must have just left it in there.

Reply:
Got it, except that loop only has a single iteration- it sounds like you want `for x in range(0, 2):`.

Reply:
Looks like you corrected this in https://github.com/cisagov/gophish-tools/pull/40/commits/3bceac856e75b598686b2aef04c4ce05f86790d8 - thanks!

Reply:
See 3bceac856e75b598686b2aef04c4ce05f86790d8 for changes to this file.